rhc-ose-ansible/roles/registry/tasks/auth_layer_nginx.yaml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Install Nexus Repository
+  yum:
+    name: "{{ nexus_repo_url }}"
+    state: present
+
+- name: Install NGinx
+  yum: name=nginx state=latest
```

Comment:
May have a problem down the road using yum and may want to change that yum module to follow this: https://github.com/openshift/openshift-ansible/blob/master/docs/best_practices_guide.adoc#yum-and-dnf

basically looking like this:

# tasks.yml
- name: Install NGinx
  action: "{{ ansible_pkg_mgr }} name=nginx state=latest"
  register: install_result